File: 0027-remove-element/0027-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        count = 0

        for i in range(len(nums) - 1, -1, -1):
            if nums[i] == val:
                nums.pop(i)
                count += 1
        return len(nums)
        

File: 0027-remove-element/0027-remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int count = 0;
        std::replace(nums.begin(), nums.end(), val, 1000000);
        sort(nums.begin(), nums.end());

        for(int i = 0; i < nums.size(); i++){
            std::cout << nums[i] << ", ";
            if(nums[i] == 1000000){
                break;
            }
            else{
                count++;
            }
        }
        return count;
    }
};